src/Features/LanguageServer/ProtocolUnitTests/ProtocolConversionsTests.cs

```diff
@@ -6,13 +6,63 @@
 using System.Linq;
 using Microsoft.CodeAnalysis.Text;
 using Microsoft.VisualStudio.LanguageServer.Protocol;
+using Roslyn.Test.Utilities;
 using Xunit;
 using Range = Microsoft.VisualStudio.LanguageServer.Protocol.Range;
 
 namespace Microsoft.CodeAnalysis.LanguageServer.UnitTests
 {
     public class ProtocolConversionsTests
     {
+        [ConditionalTheory(typeof(WindowsOnly))]
```

Comment:
What about the "git://" format?

Reply:
That's already a URL. The issue is only when converting local paths (e.g. `C:\a\b.cs` or `\\server\x`) to URL.

Reply:
Added tests for git://